test/integration/local/payment-request-validation.test.js

```diff
@@ -4,7 +4,7 @@ let paymentRequest
 describe('payment request validation', () => {
   beforeEach(async () => {
     paymentRequest = {
```

Comment:
Move into `mockPaymentRequest` module -- used throughout and can easily be adjusted for specific use-cases.

Also means we're testing on a defined and close to 'real' dataset

Reply:
Seems sensible, though currently doesn't have much value outside this test suite as others are testing pre-enrichment cases and processing steps.